medicines/pars-upload/src/review_submission.js

```diff
@@ -63,7 +63,12 @@ export const ReviewSubmission = ({
         }
       })}
 
-      <Button type="button" onClick={submit}>
+      <Button
+        type="button"
+        onClick={() => {
+          submit(null)
```

Comment:
This has potential gotcha written all over it if someone removes the `null` later. Maybe we could have a test that submits twice?

Reply:
Yeah, that's definitely a good idea. If I have time once the other PARs stuff is done I'll try and get round to it but if not I'll just merge this as is 😅